libr/anal/p/anal_6502_cs.c

```diff
@@ -199,7 +200,8 @@ static int set_reg_profile(RAnal *anal) {
 		"gpr	N	.1	.31	0\n"
 		"gpr	sp	.8	4	0\n"
 		"gpr	pc	.16	5	0\n";
-	return r_reg_set_profile_string (anal->reg, p);
+	r_strbuf_init_const (&sb, p, strlen (p));
+	return r_reg_set_profile_string (anal->reg, &sb);
```

Comment:
i dont see any benefit on this change of function signture, the _string should be kept as const char *